features/step_definitions/your_results_steps.rb

```diff
@@ -99,6 +99,11 @@
   end
 end
 
+Then(/^my results should all read:$/) do |table|
+  headings = table.raw.flatten
+  expect(your_results_page.results_period_headings[0..2].map(&:text)).to eq(headings)
```

Comment:
@margOnline Is there a way to make this flow with English-like syntax?

i.e.
`expect(your_results_page.result_table_heading).to eq(headings)`